explore-analyze/alerts-cases/alerts/maintenance-windows.md

```diff
@@ -52,6 +48,15 @@ When you create a maintenance window, you must provide a name and a schedule. Yo
 
 By default, maintenance windows affect all categories of rules. The category-specific maintenance window options alter this behavior. For the definitive list of rule types in each category, refer to the [get rule types API](https://www.elastic.co/docs/api/doc/kibana/group/endpoint-alerting).
 
+::::{note}
+
+{applies_to}`stack: removed 9.2` {applies_to}`serverless: removed` The option to specify rule categories for a maintenance window is no longer available. Maintenance windows apply to all rule types.
+
+Note that existing maintenance windows are still applied to rule categories that were specified. However, if you edit a maintenance window after upgrading or using the latest version of {{serverless-short}}, specified rule categories are removed and the maintenance window will be applied to all rules types.
```

Comment:
Sorry my bad for not pointing out earlier, that removing category will cause MW to apply to all solutions, not all rule types.
nit: `specified rule categories are removed and the maintenance window will be applied to all rules types.` -> `specified rule categories are removed and the maintenance window will be applied to rules in all solutions.`